tiledb/sm/filesystem/azure.cc

```diff
@@ -115,15 +108,22 @@ Status Azure::init(const Config& config, ThreadPool* const thread_pool) {
     blob_endpoint = std::string(tmp);
   }
   if (blob_endpoint.empty()) {
-    LOG_WARN("The 'vfs.azure.blob_endpoint' option is not specified.");
-  }
-  if (!blob_endpoint.empty() &&
-      !(utils::parse::starts_with(blob_endpoint, "http://") ||
-        utils::parse::starts_with(blob_endpoint, "https://"))) {
+    if (!account_name.empty()) {
+      blob_endpoint = "https://" + account_name + "blob.core.windows.net";
```

Comment:
Should this be `"https://" + account_name + ".blob.core.windows.net"`?

Reply:
Oops, thanks!